src/en/installconfig-commission-nodes.md

```diff
@@ -17,19 +17,30 @@ require changing. However, it can be configured in the web UI in the 'Settings'
 page.
 
 To commission a node, select it and choose 'Commission' under the 'Take action'
-dropdown menu (orange button). Its status will change to *Commissioning*. 
+dropdown menu (orange button).
+
+![commission](./media/installconfig-commission-nodes__commission.png)
+
+You have the option of selecting some extra parameters (checkboxes). Then
+finalize the directive by hitting 'Go'.
+
+![commission go](./media/installconfig-commission-nodes__commission-go.png)
+
+While a node is commissioning its status will change to *Commissioning*. 
 
 See [MAAS CLI](./manage-cli-common.html#commission-all-machines) for how to
-commission all machines with the 'Ready' status.
+commission all machines with the 'New' status.
 
 !!! Note: If your node has more than one network interface you may need to tell
 MAAS which one to use. Do this by marking it *Broken* and proceeding similarly to
```

Comment:
I know this isn't part of your changes but this note could do with a grammar overhaul 'proceeding similarly to what is shown'?